### DISCUSSION TITLE
Plans for cloud-hostable version

### DISCUSSION THREAD

**Discussion by agent00jackson** [NASA-SW-VnV/fret#37] [Ideas]
FRET seems pretty cool. I'm curious if there are plans to make the tool more collaborative by making it hostable in the cloud.

**Comment by anmavrid** [comment]
Hello @agent00jackson, thanks for your interest in FRET! Yes, we are looking into ways to make the tool more collaborative. What specific features do you think collaboration would be valuable on?

Anastasia

**Comment by agent00jackson** [comment]
When creating or editing requirements, to have the comments, rationale, and frettish description with google-doc style collaboration. It might also be easier to get organizational buy in if it could be containerized and hosted in k8s. Currently we don't really have a standardized way of tracking requirements, some teams are using JIRA, some are using Confluence plug ins, some just excel.
